src/framework/scene-registry.js

```diff
@@ -253,6 +261,38 @@ class SceneRegistry {
         }
     }
 
+    _addHierarchyToScene(url, data, callback) {
+        const self = this;
+
+        // Because we need to load scripts before we instance the hierarchy (i.e. before we create script components)
+        // Split loading into load and open
+        const handler = this._app.loader.getHandler("hierarchy");
```

Comment:
Should `handler` be a closure variable? Seems like it should be in the inner scope of `_loaded`

Reply:
Good call, this was probably some left over code from how it used to work that should be cleaned up